public/js/index.js

```diff
@@ -2016,3 +2016,7 @@ function initFilterBranchTagDropdown(selector) {
         });
     });
 }
+
+$(".commit-button").click(function() {
+    $(this).next().toggle();
```

Comment:
I'm not a JS expert, but this seems fragile. What if we add a DOM element between the commit button and the commit body? Perhaps it would be better to select for the `commit-body` class?

Reply:
I tried doing

```js
$(".commit-button").click(function() {
    $("commit-body").toggle();
```

But that opens and closes all of the commit messages, not just the one selected. All the documentation I could find from both JQuery themselves and online used this method, otherwise I think we'd have to use one of those JQuery-Toggle libraries.

Reply:
Try:
```js
$(".commit-button").on('click', function() {
  $(this).parent().find('.commit-body').toggle();
});
```

Reply:
Thanks, I'll give it a go later today.